pkg/debug/checkups/files.go

```diff
@@ -0,0 +1,78 @@
+package checkups
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"runtime"
+	"strings"
+
+	"github.com/kolide/launcher/pkg/agent/types"
+)
+
+type filesCheckup struct {
+	k       types.Knapsack
+	status  Status
+	summary string
+	data    map[string]any
+}
+
+func (fc *filesCheckup) Data() map[string]any  { return fc.data }
+func (fc *filesCheckup) ExtraFileName() string { return "" }
+func (fc *filesCheckup) Name() string          { return "Notable Files" }
```

Comment:
I think the name is preexisting, but since this check exists to determine whether there could be a secondary/separate osquery installation that interferes with the Kolide/launcher installation, maybe a good time to rename this to "Non-launcher osquery installation" or "Unexpected osquery config files" or something else more specific?

Reply:
that makes sense to me! maybe `Osquery Conflicts`? just trying to think of something shortish since these turn into the dir names